misk/src/test/kotlin/misk/config/HttpClientsConfigBackwardsCompatibilityTest.kt

```diff
@@ -23,7 +23,7 @@ class HttpClientsConfigBackwardsCompatibilityTest {
                 connectTimeout = Duration.ofSeconds(41),
                 readTimeout = Duration.ofSeconds(42),
                 writeTimeout = Duration.ofSeconds(43)
-            )
+            ).applyDefaults(HttpClientsConfig.httpClientConfigDefaults)
```

Comment:
Wasn't sure if it makes sense to just hardcode the default values in the test, or apply global defaults. Might revisit.